authorization-common/src/main/java/com/sequenceiq/authorization/utils/EventAuthorizationUtils.java

```diff
@@ -0,0 +1,56 @@
+package com.sequenceiq.authorization.utils;
+
+import static com.sequenceiq.authorization.resource.AuthorizationResourceAction.DESCRIBE_ACTION_PREFIX;
+import static com.sequenceiq.authorization.resource.AuthorizationResourceAction.GET_ACTION_PREFIX;
+import static com.sequenceiq.cloudbreak.util.NullUtil.throwIfNull;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
+import com.sequenceiq.authorization.service.CommonPermissionCheckingUtils;
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
+
+@Component
+public class EventAuthorizationUtils {
+
+    private CommonPermissionCheckingUtils permissionCheckingUtils;
+
+    public EventAuthorizationUtils(CommonPermissionCheckingUtils permissionCheckingUtils) {
+        this.permissionCheckingUtils = permissionCheckingUtils;
+    }
+
+    public void checkPermissionBasedOnResourceTypeAndCrn(Collection<EventAuthorizationDto> eventAuthorizationDtos) {
```

Comment:
instead of Collection of new dtos, you can use Map, then Dto would not needed

Reply:
is using dto cause any issue?

Reply:
no, it just seems overkill to create a new object for a string pair, probably we will not use it in other cases and we will not extend it